mobs/nakki_v2/ai_states/NakkiEmergeOnTriggerState.cs

```diff
@@ -0,0 +1,93 @@
+using Godot;
+using System;
+using System.Linq;
+
+using Jakojaannos.WisperingWoods.Util.Editor;
+
+namespace Jakojaannos.WisperingWoods;
+
+[Tool]
+public partial class NakkiEmergeOnTriggerState : NakkiAiState {
+	public override string[] _GetConfigurationWarnings() {
+		return (base._GetConfigurationWarnings() ?? Array.Empty<string>())
+			.Union(this.CheckCommonConfigurationWarnings())
+			.ToArray();
+	}
+
+	[Export]
+	[MustSetInEditor]
+	public NakkiAiState? EnterStateAfterEmerge {
+		get => this.GetNotNullExportPropertyWithNullableBackingField(_enterStateAfterEmerge);
+		set => this.SetExportProperty(ref _enterStateAfterEmerge, value, notifyPropertyListChanged: true);
+	}
+	private NakkiAiState? _enterStateAfterEmerge;
+
+	[Export] private float _emergeAnimationSpeed = 3.0f;
+	[Export] private float _emergeDelay = 3.0f;
```

Comment:
I prefer having such stats/configs as public properties.

While encapsulation is nice, I'd prefer open access to configuration by-default. It's a fine line, so I can't really argue better what should be public and what private, but I think have a bit more "configuration"/"stats" variables feel to them, so having them as public might be fine.

As you said, exported private is a bit of an oxymoron, or something 🤷‍♀️

Or yeah, it feels a bit wonky if something somewhere is able to adjust these at runtime, at any moment. On the other hand, there is likely little harm from doing so. Thus, I don't see a reason to prohibit other classes from accessing these. It could be a powerful tool for simple one-off level scripts to be able to adjust stats like these, if needed for some quick hack.